boards/arm/rcar_m3ulcb/rcar_m3ulcb_cr7.yaml

```diff
@@ -0,0 +1,16 @@
+identifier: rcar_m3ulcb_cr7
+name: Cortex r7 for Renesas M3ULCB
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+supported:
+  - i2c
+  - can
+  - gpio
+  - clock_control
+  - uart
+testing:
+  ignore_tags:
+    - isotp
```

Comment:
How come you explicitly ignore isotp?

Reply:
We have an issue on R-Car CAN loopback implementation:
Interrupt for an Rx message is raised before having the Tx done interrupt,
that confuse the isotp state machine.